### ISSUE TITLE
Upgrade packages flagged in Dependabot alerts

### ISSUE THREAD

**Issue by rhigman** [thoth-pub/thoth#388]

**Comment by rhigman** [comment]
Alert 1 will be fixed by #397 (yew 0.19.3 removes anymap dependency).
Alerts 21 and 5 blocked on release of failure >0.1.8.

**Comment by rhigman** [comment]
Alert 23 has been withdrawn.
Alert 31 is blocked on release of owning_ref >0.4.1. The package is only used by lock_api 0.1.5, so this is another issue which is blocked by wasm-pack dependencies, as above.
Alerts 27, 28, 29, 30 require upgrading to juniper 0.15.10. We are currently on 0.14.2 and there will be some breaking changes. Tracked at https://github.com/thoth-pub/thoth/issues/457.

**Comment by ja573** [comment]
Most have now been closed with #509 

Ignoring yew/wasm alerts as they will be dropped with the GUI redesign, the only alert left is related to `chrono`'s dependency on `time`, which will be dropped in the [next version](https://github.com/chronotope/chrono/issues/1073)
